src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -45,13 +54,16 @@ def get_df_data():
 
         # add to data_list if some kinds of question, even if unanswered
         if len(pageid) >= 4 and status not in EXCLUDE_STATUS:
-            data_list.append({
-                'id': item['id'],
-                'title': item['name'],
-                'status': status,
-                'pageid': pageid,
-                'url': url,
-            })
+            for i, name in enumerate(names):
+                data_list.append({
+                    # ids must be unique, and the first name is the original name, which means that
+                    # any subsequent names are duplicates
+                    'id': f'{item["id"]}{i or ""}',
```

Comment:
a bit of magic here to make sure that the ids are unique. This has the nice effect of leaving the prexisting values in the index as are, and adding new ids for alternative phrasings